recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -169,6 +170,14 @@ def upload_or_check(
                 os.path.join(conda_build.config.croot, conda_build.config.subdir))
         ]
     )
+    DEFAULT_OS=set([subdir.split("-")[0] for subdir in DEFAULT_SUBDIRS])
+    for subdir in os.listdir(conda_build.config.croot):
+        if not subdir.split("-")[0] in DEFAULT_OS:
+            continue
```

Comment:
What is this if statement catching?

Reply:
There are lots of folders in `<conda-root>/conda-bld`. This removes anything that doesn't look like a subdir